scripts/pre_push_hook.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
@anmolshkl sorry, what decision?

Also @prasanna08 -- re the parity between the files here and in Oppia, I only meant to add the "keep in sync" comment if the files should actually be kept in sync. However if they are intended to evolve independently then we don't need that comment. Perhaps that latter option might be better if we really want to keep the repos separate.